pkg/rpc/response/result/contract_state.go

```diff
@@ -11,7 +13,7 @@ import (
 type ContractState struct {
 	Version     byte                      `json:"version"`
 	ScriptHash  util.Uint160              `json:"hash"`
-	Script      []byte                    `json:"script"`
+	Script      string                    `json:"script"`
```

Comment:
Same thing here, it's way more convenient for the user of client API to get raw script, rather than hex that it would need to decode.

Reply:
Done, but also had to add marshaller for ContractState in order to match API: https://docs.neo.org/docs/en-us/reference/rpc/latest-version/api/getcontractstate.html